### ISSUE TITLE
A child process terminated abruptly, the process pool is not usable anymore

### ISSUE THREAD

**Issue by Graydenn** [HexCodeFFF/mediaforge#33]
**Describe the bug**
Tried to create meme, failed with this error

**To Reproduce**
Typed `$motivate BAD | THIS IS REALLY BAD`

**Expected behavior**
Meme should be created

**Screenshots**
[Screenshot](https://imgur.com/a/vkJPMpo)

**Traceback**
[The traceback](https://files.catbox.moe/5lhabu.txt)

**Additional context**
n/a

**Comment by HexCodeFFF** [comment]
This error doesn't have to do specifically with the $meme command but it stems from more general functions for managing rendering. Specifically if there's an error in the code that starts up the renderers, it wont appear until later for... some reason. In my testing I got an error in the initializer due to a wrong version of the chrome driver on windows.
I tried to write some better handling code but because of the strange way `ProcessPoolExecutor` works I'd have to trade off functionality so I'll just close this for now.

Sidenote: "To Reproduce" means a set of instructions that can _consistently_ cause the error to appear so I can have a better look at it with debugging tools, not just the command that caused the error.
